package.json

```diff
@@ -23,19 +23,19 @@
   "dependencies": {
     "binaryen": "102.0.0-nightly.20211028",
     "long": "^4.0.0",
-    "source-map-support": "^0.5.20",
-    "ts-node": "^6.2.0"
+    "ts-node": "^10.4.0"
```

Comment:
When `ts-node` is removed in releases, it seems `source-map-support` is not available anymore

Reply:
Yes, but it should be installed. When script replace `index.js` it will remove this installation step anyway. Or I don't understand something?